src/target/riscv/riscv-013.c

```diff
@@ -1678,7 +1690,7 @@ static int examine(struct target *target)
 		LOG_DEBUG("Detected %d harts.", dm->hart_count);
 	}
 
-	r->current_hartid = target->coreid;
+	info->index = target->coreid;
```

Comment:
Do we need this assignment? The value is already assigned above on line 1574.

Reply:
Fixed. Thanks.